docker/api/container.py

```diff
@@ -174,6 +175,14 @@ def logs(self, container, stdout=True, stderr=True, stream=False,
                 if tail != 'all' and (not isinstance(tail, int) or tail <= 0):
                     tail = 'all'
                 params['tail'] = tail
+            if utils.compare_version('1.19', self._version) >= 0:
+                if isinstance(since, datetime):
```

Comment:
I think normally we want to raise an error if someone is using an option that isn't supported by the API version.  It would be nice if the server did this, but I don't think that's the case most of the time.